src/Authentication/Controllers/SystemUserController.cs

```diff
@@ -144,7 +144,7 @@ public async Task<ActionResult<SystemUser>> CreateSystemUser(string reporteeOrgN
         /// Replaces the values for the existing system user with those from the update 
         /// </summary>
         /// <returns></returns>
-        [Authorize]
+        [Authorize(Policy = AuthzConstants.POLICY_ACCESS_MANAGEMENT_WRITE)]
```

Comment:
i'm unsure about the usage of access management policy on system user api. @TheTechArch @jonkjetiloye I assume we need to define different policy for system user, Is that right ?

Reply:
was resolved